src/core/server/logging/layouts/json_layout.ts

```diff
@@ -41,8 +42,20 @@ export class JsonLayout implements Layout {
     };
   }
 
+  private static mergeLogMeta(log: Ecs, meta: LogMeta): Ecs {
+    // Paths which cannot be overridden via `meta`.
+    const RESERVED_PATHS = ['ecs', '@timestamp', 'message', 'log.level', 'log.logger'];
+
+    for (const path of RESERVED_PATHS) {
+      unset(meta, path);
+    }
```

Comment:
There were a few which were not previously overridden, but I've updated it now based on Mikhail's suggestion below so that core's values will now always overwrite user-provided ones